Project architecture/MVVM.md

```diff
@@ -136,8 +136,3 @@ At this point, the concept of MVVM and how to implement it should be much cleare
 
 A textbook example of our proposal of MVVM architecture can be found on this [GitHub repo](https://github.com/infinum/Android-MvvM-Example). In the README, you will find all the details for a base setup and using Android architecture components.
```

Comment:
Mvvm Example is also private. Will we make it public or remove it from here as well?

Reply:
Yeah, we're currently talking about making the MVVM project a public one. If we decide against this, I'll revisit the described issue 🙂